packages/edit-post/src/plugins/index.js

```diff
@@ -16,11 +16,16 @@ import ToolsMoreMenuGroup from '../components/header/tools-more-menu-group';
 import WelcomeGuideMenuItem from './welcome-guide-menu-item';
 import BlockPatterns from './block-patterns';
 
+registerPlugin( 'edit-post-block-patterns', {
```

Comment:
maybe `core/edit-post/block-patterns`  though I see that the other plugin doesn't respect this convention.

Reply:
Have you considered keeping it in its own module similar to Block Directory? This way you can prevent enqueueing the script when the experiment is disabled.

Reply:
That might make sense, if it allows us to keep patterns as a plugin even as we move it away from `PluginSidebar`, as long as we use the right kind of Slots to integrate patterns well.